src/main/java/ua/okwine/productexpirationdate/exceptions/NotExistedSupplierId.java

```diff
@@ -0,0 +1,12 @@
+package ua.okwine.productexpirationdate.exceptions;
+
+public class NotExistedSupplierId extends RuntimeException {
+
+    public NotExistedSupplierId(String message) {
```

Comment:
Create constant with string "Supplier not found with id: " and change input value to id(UUID).

Reply:
ok